### PR TITLE
new(mp): New VerneMQ Restapi Monitoring Procedure [FR]

### PR DESCRIPTION
**Pull request by psamecentreon**

### CODE REVIEW

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+Plus d'informations sur l'API HTTP de VerneMQ sur : https://docs.vernemq.com/administration/http-administration#managing-api-keys
+
+## Installation
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online IMP Licence & IT-100 Editions-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources VerneMQ :
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:79]: Uppercase C ;)

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+```bash
+yum install centreon-plugin-Applications-Vernemq-Restapi.noarch
+```
+
+2. Sur l'interface Web de Centreon, installer le Plugin-Pack *VerneMQ Restapi* depuis la page "Configuration > Plugin packs > Manager"
+
+<!--Offline IMP License-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources VerneMQ :
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:89]: Uppercase C

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+## Prérequis
+
+Un certain nombre de distributions fournissent VerneMQ, y compris des paquets binaires pré-construits.
+Le support de ces compilations, s'il y en a, est fourni par le fournisseur de la distribution associée.
+Leur cycle de publication peut être en retard par rapport aux versions sources de VerneMQ.
+
+De plus amples informations sont disponibles sur la documentation officielle du VerneMQ : https://docs.vernemq.com/getting-started
+
+L'API HTTP de VerneMQ est activée par défaut et installe un gestionnaire HTTP sur `http://localhost:8888/api/v1`.
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:63]: Cf EN review

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+La commande ci-dessus contrôle les sessions de VerneMQ via la Restapi (```--mode=sessions```).
+Le Plugin utilise l'_api-key_ qui correspond au Token VerneMQ (```--api-key='12342939495003'```)
+et il se connecte à l'hôte _myvernemq.com_ (```--hostname='myvernemq.com'```) 
+sur le port _8888_ (```--port='8888'```) utilisant le protocol _http_ (```--proto='http'```).
+
+Toutes les options et leur utilisation peuvent être consultées avec le paramètre ```--help``` ajouté à la commande:
+
+```bash
+/usr/lib/centreon/plugins/centreon_vernemq_restapi.pl --plugin=apps::mq::vernemq::restapi::plugin \
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:148]: cf EN review

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+#### ```UNKNOWN: 501 Protocol scheme 'connect' is not supported |``` 
+
+Suite à la mise en place du proxy, j'obtiens le message suivant ```UNKNOWN: 501 Protocol scheme 'connect' is not supported |```
+Cela signifie que le protocole de connexion au proxy n'est pas supporté par la libraire *curl* utlisée par défaut par le Plugin Centreon.
+Cette erreur peut être résolue en utilisant le backend HTTP *curl*. Pour ce faire, ajoutez l'option ```--http-backend='curl'``` à la commande.
+
+### Comment puis-je supprimer les perfdatas *count* dans le cas où je ne souhaite vérifier qu'une seule application ?
+
+Le Plugin permet de filtrer sur un ou plusieurs éléments mais permet également de récupérer l'ensemble des éléments si aucun filtre n'est spécifié.
+De ce fait, des perfdatas "globales" sur les statistiques des objets sont ajoutées par défaut. Il est possible de supprimer ces données de performance en appliquant le filtre suivant: ```--filter-perfdata='^$'```.
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:169]: cf EN review

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md (hunk 1 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md]: Done ^^

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md (hunk 2 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md]: Done ^^

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md (hunk 3 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md]: Yes, it's OK now

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md (hunk 5 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md]: Yes, it's OK now

File: fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md (hunk 4 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-vernemq-restapi.md]: Yes, it's ok

**Review by Sims24 — APPROVED** [review]

**Pull request merged**